stream-chat-android-client/src/main/java/io/getstream/chat/android/client/events/ChatEvent.kt

```diff
@@ -265,6 +265,15 @@ public data class NotificationMarkReadEvent(
     @SerializedName("unread_channels") val unreadChannels: Int?
 ) : CidEvent(), UserEvent
 
+public data class MarkAllReadEvent(
+    override val type: String = "",
+    @SerializedName("created_at") override val createdAt: Date,
+    override val user: User,
+    @SerializedName("unread_count") val unreadCount: Int = 0,
+    @SerializedName("total_unread_count") val totalUnreadCount: Int = 0,
+    @SerializedName("unread_channels") val unreadChannels: Int = 0
```

Comment:
We don't use default parameter values for any other event, why are we using them here? Gson will ignore them anyway, which might be confusing.

Reply:
When we create a MarkAllRead event after pulling the timestamp from syncState in setUser, it seemed cleaner than dropping a magic 0 in for three parameters.